index.html

```diff
@@ -291,7 +300,32 @@ <h5 class="modal-title">Registration successful</h5>
       </div>
     </div>
   </div>
-</div>
+
+<!-- Cvent registration modal -->
+    <div class="modal fade" id="registrationModal" role="document">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title">Please complete registration below</h5>
+        </div>
+        <div class="modal-body">
+          <div class="embed-responsive embed-responsive-1by1">
+            <iframe class="embed-responsive-item" src="http://www.cvent.com/events/lsst-desc-winter-collaboration-meeting/event-summary-1672e3a8972a4d608897b80dd32f5ce4.aspx?fqp=true" allowfullscreen></iframe>
```

Comment:
I wonder if there's a way to pre-fill the cvent form. Probably not...

Reply:
@EiffL can you see if making this `https` would solve the unsafe script issue?
```suggestion
            <iframe class="embed-responsive-item" src="https://www.cvent.com/events/lsst-desc-winter-collaboration-meeting/event-summary-1672e3a8972a4d608897b80dd32f5ce4.aspx?fqp=true" allowfullscreen></iframe>
```